moveit_ros/planning/planning_pipeline/include/moveit/planning_pipeline/planning_pipeline.h

```diff
@@ -57,6 +57,79 @@ namespace planning_pipeline
 class PlanningPipeline
 {
 public:
+  /**
+   * @brief      This class contains the rcl interfaces for easier testing
+   */
+  class MiddlewareHandle
+  {
+  public:
+    /**
+     * @brief      Destroys the object.
+     */
+    virtual ~MiddlewareHandle() = default;
```

Comment:
IMHO destructor is a fundamental C++ building block that does not require documentation.